src/corelibs/U2Core/src/io/IOAdapterTextStream.cpp

```diff
@@ -0,0 +1,204 @@
+/**
```

Comment:
I think, we don't have a rule us to be punched for each new file each of us has add to the project. Cuz I have no Idea why several classes are concentrated in a single ".cpp/.h" pair of files so often. I offer to split these classes - one class one ".cpp/.h" pair of files.

Reply:
I do not support this level of separation: 1 file per 1 class.

Multiple very related classes can live within 1 file if the overall file size is small.